### PR TITLE
Add request body matching

### PR DESCRIPTION
**Pull request by therve**
We didn't check body previously. This also removes old interactions when
recording.

### CODE REVIEW

File: features/support/env.rb [hunk 1]
```diff
@@ -27,11 +27,11 @@
 
 module RecordMode
   def self.true(config)
-    config.default_cassette_options = { :record => :all }
+    config.default_cassette_options = { :record => :all, :allow_unused_http_interactions => false, :clean_outdated_http_interactions => true, :re_record_interval => 3600}
```
Comment (jirikuncar) [inline features/support/env.rb:30]:
```suggestion
    config.default_cassette_options = { :record => :all, :allow_unused_http_interactions => false, :clean_outdated_http_interactions => true, :re_record_interval => 3600 }
```

File: features/support/env.rb [hunk 2]
```diff
@@ -27,11 +27,11 @@
...
   end
 
   def self.false(config)
-    config.default_cassette_options = { :record => :none }
+    config.default_cassette_options = { :record => :none, :allow_unused_http_interactions => false}
```
Comment (jirikuncar) [inline features/support/env.rb:34]:
```suggestion
    config.default_cassette_options = { :record => :none, :allow_unused_http_interactions => false }
```

File: features/support/hooks.rb [hunk 3]
```diff
@@ -35,7 +35,8 @@
 end
 
 Around do |scenario, block|
-  VCR.use_cassette(scenario.location.file.chomp('.feature') + "/" + scenario.name.gsub(/[^A-Za-z0-9]+/, '-')[0..100], :record_on_error => false) do |cassette|
+  VCR.use_cassette(scenario.location.file.chomp('.feature') + "/" + scenario.name.gsub(/[^A-Za-z0-9]+/, '-')[0..100], :record_on_error => false, :match_requests_on => [:host, :path, :uri, :body]) do |cassette|
```
Comment (NouemanKHAL) [inline features/support/hooks.rb:38]:
```suggestion
  VCR.use_cassette(scenario.location.file.chomp('.feature') + "/" + scenario.name.gsub(/[^A-Za-z0-9]+/, '-')[0..100], :record_on_error => false, :match_requests_on => [:method, :host, :path, :uri, :body]) do |cassette|
```
Matching with the method too maybe?

**Review by NouemanKHAL — APPROVED** [review]

**Pull request merged**